### ISSUE TITLE
How to move test pane

### ISSUE THREAD

**Issue by adamboche** [pghilardi/atom-python-test#10]
Is it possible to move the test text-box around? For example, I'd like to have it to the right of my code, rather than below. Ideally, it could be just like a regular buffer/pane, and could be moved around, kept in a tab, and switched to/from.

**Comment by pghilardi** [comment]
Adam,

This is a bit more complicated than it seems. I'm using a simple pane view and I need to investigate a better solution to be able to change the position of it. Actually, my idea is to build a whole new graphical interface to this plugin, supporting this requirement and more. 

Maybe, as a simple solution, I may enable the user to configure if he wants to show the pane on bottom or right. This would be a simple change. What do you think?

**Comment by adamboche** [comment]
Yep, that sounds like a good solution! It'd be very nice to have the option
to put it on any of four edges (top, bottom, left, right) and in any of the
four quadrants. Thanks for asking!
